### DISCUSSION TITLE
Add GeoServer Vector Tile to MapboxVectorTileCatalogItem

### DISCUSSION THREAD

**Discussion by mdchuc** [TerriaJS/terriajs#6150] [Q&A]
Hi,

I am trying to add a polygon vector tile layer built by GeoServer to TerriaMap (terriajs v8.1.21). I used the following config in the wwwroot/init/simple.json file:

```
{
    "homeCamera": {
      "north": 15,
      "east": 104,
      "south": 11,
      "west": 96
    },
    "catalog": [
      {
        "type": "mvt",
            "name": "Land QOL",
            "url": "http://wwww.localhost:8080/geoserver/gwc/service/tms/1.0.0/satreps:landqol@EPSG%3A900913@pbf/{z}/{x}/{y}.pbf",
            "lineColor": "hsl(180,10%,30%)",
            "fillColor": "hsl(180,10%,30%)",
            "layer": "satreps:landqol"
      }
    ],
    "baseMapId": "basemap-bing-aerial-with-labels",
    "previewBaseMapId": "basemap-positron"
  }
```

My data is in EPSG: 4326 projection.  _**satreps**_ is store name and **_landqol_** is layer name, according to GeoServer management style.
I tried to visualized the data using this url in mapbox gl and it worked. 
But in TerriaMap, it throws 404 error. I am not sure how should I config to be able to load the vector tiles?

Thanks,

**Comment by nf-s** [comment]
Hi @chuc92man 

Sorry I am not familiar with the GeoServer Vector Tiles extension.
Can you please provide some of the URLs that Terria is attempting to request?
These should be printed to console (if 404s are occurring) - or they can be seen in the Network tab in Developer Tools

Thanks,
Nick
